chandra_aca/aca_image.py

```diff
@@ -309,6 +311,241 @@ def col0(self):
     def col0(self, value):
         self.meta['IMGCOL0'] = np.int64(value)
 
+    @classmethod
+    def _read_flicker_cdfs(cls):
+        """Read flickering pixel model cumulative distribution functions
+        and associated metadata.  Set up class variables accordingly.
+
+        The flicker_cdf file here was created using:
+        /proj/sot/ska/www/ASPECT/ipynb/chandra_aca/flickering-pixel-model.ipynb
+
+        """
+        from astropy.io import fits
+
+        filename = Path(__file__).parent / 'data' / 'flicker_cdf.fits.gz'
```

Comment:
Does setup.py need a modification to actually install this file in package data?

Reply:
Yes, good catch, done.